sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -27,6 +27,7 @@
 import AzureCommunication
 import AzureCore
 import Foundation
+import AzureCommunicationSignaling
```

Comment:
As discussed during the arch meeting, this functionality should simply be built into the AzureCommunicationChat convenience layer instead of being a separate module.